tests/helpers.py

```diff
@@ -347,22 +347,28 @@ def create_pvc(
     return ocs_obj
 
 
-def create_multiple_pvcs(sc_name, namespace, number_of_pvc=1, size=None):
```

Comment:
not related to this PR specifically but this function and pvc.delete_pvcs() should be under the same file for consistency